contentcuration/contentcuration/viewsets/tree.py

```diff
@@ -81,7 +81,9 @@ def list(self, request, *args, **kwargs):
             raise MissingRequiredParamsException(
                 "channel_id query parameter is required but was missing from the request"
             )
-        root = get_object_or_404(ContentNode, channel_main=channel_id)
+        root = ContentNode.objects.filter(Q(channel_trash=channel_id) | Q(channel_main=channel_id)).first()
```

Comment:
This will lead to unpredictable behaviour, depending on database ordering - this is only ever returning a single node, so we are either getting nodes from the main tree or nodes from the trash tree, and it's not predictable which.

Might be better to add a trash get param that conditionalizes this behaviour?